### PR TITLE
feat(chart): support livenessProbe and readinessProbe

### PR DESCRIPTION
**Pull request by maximee-leroyy**
## Description

Adds a livenessProbe and readinessProbe so that it restarts the pod if it hangs up for whatever reason

## Checklist

 - [x] I made sure to update `./CHANGELOG.md`.
 - [x] I made sure to add any docs changes required for my change (including release notes).
 - [x] My change is adequately tested.
 - [ ] I updated `DEVELOPING.md` with any any special dev tricks I had to use to work on this code efficiently.
 - [ ] I updated `TELEMETRY.md` if I added, changed, or removed a metric name.
 - [ ] Once my PR is ready to have integration tests ran, I posted the PR in #telepresence-dev in the datawire-oss slack so that the "ok to test" label can be applied.

### CODE REVIEW

File: charts/telepresence/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Change Log
 
-### 2.6.8 (TBD)
+### 2.9.0
+
+- Feature: add `livenessProbe`, `readinessProbe` value.
+
```
Comment (thallgren) [inline charts/telepresence/CHANGELOG.md]: This must be moved to the CHANGELOG.md at the repository root. Also, please make the entry more verbose. Where is it added? For what purpose? What pod is it for?

The CHANGELOG.md here is no longer maintained (and yes, we should remove it).

**Review by thallgren — CHANGES_REQUESTED** [review]
Thanks for providing this. It looks besides my comment about the changelog.

You'll need to sign off on the commit to pass our DCO check.

File: charts/telepresence/CHANGELOG.md (hunk 1 above)
Comment (maximee-leroyy) [reply charts/telepresence/CHANGELOG.md]: okay thanks for the review, i have done the changes!

**Comment by maximee-leroyy** [comment]
> Thanks for providing this. It looks besides my comment about the changelog.
> 
> You'll need to sign off on the commit to pass our DCO check.

thanks for the review @thallgren, it should be good now!

**Review by thallgren — APPROVED** [review]

**Pull request merged**